### DISCUSSION TITLE
Erro 404 Not Found

### DISCUSSION THREAD

**Discussion by WillianCamargo01** [RamonSilva20/mapos#2051] [General]
Olá, fiz a configuração porém, o sistema está retornadno a seguinte mensagem de erro no link http://localhost/mapos/localhost/mapos/index.php após a configuração com o DB e ter baixando as dependendias. 

Not Found
The requested URL was not found on this server.
---------------------
Apache/2.4.54 (Win64) OpenSSL/1.1.1p PHP/7.4.33 Server at localhost Port 80

**Comment by willph** [comment]
Você precisa corrigir a linha 42 do arquivo [mapos](https://github.com/RamonSilva20/mapos)/[application](https://github.com/RamonSilva20/mapos/tree/master/application)/[config](https://github.com/RamonSilva20/mapos/tree/master/application/config/)/config.php com a url correta. 


`http://localhost/mapos`

`$config['base_url'] = 'http://localhost/mapos';`

**Comment by WillianCamargo01** [comment]
Funcionou perfeitamente, obrigado!
Agora estou com um outro problema, não autentica o usuário administrador criado na instalação. Não sei se pode ser por conta da versão do php 7.4.33

**Reply by willph** [reply]
Essa é a versão ideal de PHP.

Qual versão está usando do MapOs? 
Aconselho baixa o código master.

**Reply by WillianCamargo01** [reply]
Estou usando a versão 4.39.0 do código master.
Já fiz a atualização do db também, não sei o porque não autentica com as credencias de adm. Verifiquei no banco também e não achei tabela gravadas com as informações de de adm da instalação. É como se a aplicação não tivesse gravado as credenciais de adm no ato da instalação.

**Comment by willph** [comment]
Se a tabela usuário não estiver preenchido, poderá criar manualmente o usuário via código SQL.
Ou precisará reinstalar, limpando o banco de dados e baixando o código novamente.
